src/infra/azure/storage/webjobs-config.ts

```diff
@@ -0,0 +1,28 @@
+import * as t from "io-ts";
+
+import * as RE from "fp-ts/ReaderEither";
+
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { pipe } from "fp-ts/lib/function";
+
+import { readFromNodeEnv } from "../../../app/env";
+
+export const WebJobsStorageConfig = t.type({
+  connectionString: t.string,
+  queueName: t.string,
+});
+
+export type WebJobStorageConfig = t.TypeOf<typeof WebJobsStorageConfig>;
+
+export const getWebJobsStorageConfigFromEnvironment: RE.ReaderEither<
+  NodeJS.ProcessEnv,
+  Error,
+  WebJobStorageConfig
+> = sequenceS(RE.Apply)({
+  connectionString: readFromNodeEnv("AzureWebJobsStorage"),
+  queueName: pipe(
+    readFromNodeEnv("QueueName"),
```

Comment:
would you put a more specific name on the environment variable? :)

Reply:
done!